pkg/kubelet/server/server_test.go

```diff
@@ -1528,11 +1529,11 @@ func TestServePortForward(t *testing.T) {
 				t.Fatalf("%d: uid: expected '%v', got '%v'", i, e, a)
 			}
 
-			p, err := strconv.ParseUint(test.port, 10, 16)
+			p, err := strconv.ParseInt(test.port, 10, 32)
```

Comment:
I can change it back but it was to match how it was being carried in all the data structures. It seems we use int32 to refer to ports. CRI and others already had int32 everywhere.

The int32 usage was done by https://github.com/kubernetes/kubernetes/issues/17095. Ports had an option and they went int32.